### PR TITLE
add book review tag https://github.com/ubtue/DatenProbleme/issues/1514

### PR DESCRIPTION
**Pull request by socheres**

### CODE REVIEW

File: Wiley Online Library.js [hunk 1]
```diff
@@ -408,7 +408,7 @@ function scrapeBibTeX(doc, url) {
 					title: 'Full Text PDF',
 					mimeType: 'application/pdf'
 				});
-			}
+			}*/
```
Comment (jriedl) [inline Wiley Online Library.js:411]: Please remove commented out code if the functionality is no longer in use.

File: Wiley Online Library.js [hunk 2]
```diff
@@ -419,8 +419,9 @@ function scrapeBibTeX(doc, url) {
 			// Make sure we pass only the DOI not the whole URL
 			doiURLRegex = /^https:\/\/doi.org\/(.*)/;
 			if (item.DOI && item.DOI.match(doiURLRegex))
-			    item.DOI = item.DOI.replace(/^https:\/\/doi.org\/(.*)/, "$1");
-
+				item.DOI = item.DOI.replace(/^https:\/\/doi.org\/(.*)/, "$1");
+			//ubtue: add article number 
+			if (item.pages && item.pages.match(/\d{5,}/)) item.pages = "article " + item.pages;
```
Comment (jriedl) [inline Wiley Online Library.js:424]: This seems error prone in the general case: Why is the pages variable used to derive and store the article number? Have the matters concerning this issue in general (e.g. https://github.com/ubtue/DatenProbleme/issues/758, https://github.com/ubtue/DatenProbleme/issues/676) been settled?

File: Wiley Online Library.js [hunk 3]
```diff
@@ -474,7 +475,7 @@ function scrapeCochraneTrial(doc, url){
 
 	processSubtitles(doc, item);
 	addBookReviewTag(doc, item);
-	validatePageCount(item);
+	//validatePageCount(item);
 
 	item.complete();
 }
```
Comment (jriedl) [inline Wiley Online Library.js:481]: Where have the missing author problems mentioned in https://github.com/ubtue/DatenProbleme/issues/1517) been solved? missing 100 fields will generally always make our QS fail.

File: Wiley Online Library.js (hunk 1 above)
Comment (socheres) [reply Wiley Online Library.js]: removed

File: Wiley Online Library.js (hunk 3 above)
Comment (socheres) [reply Wiley Online Library.js:465]: Error no longer appears

File: Wiley Online Library.js (hunk 2 above)
Comment (socheres) [reply Wiley Online Library.js]: see code.

**Comment by jriedl** [comment]
In Kürze auf nu.

**Pull request merged**